### PR TITLE
:sparkles:  Add a demo link to the readme (#102)

### PR DESCRIPTION
**Pull request by LucasProcopio**

### CODE REVIEW

**Comment by kefranabg** [comment]
Hi @LucasProcopio,

Thanks for your contribution, this seems to be great work. I'll do a review ASAP

File: package.json [hunk 1]
```diff
@@ -50,6 +50,7 @@
     "url": "https://github.com/kefranabg/readme-md-generator/issues"
   },
   "homepage": "https://github.com/kefranabg/readme-md-generator#readme",
+  "demoUrl": "https://github.com/kefranabg/readme-md-generator#-demo",
```
Comment (kefranabg) [inline package.json]: `demoUrl` is not a field that is usually used in `package.json`. The question about demoUrl should never have default value, user is free to answer the question if he wants to.

Comment (LucasProcopio) [reply package.json]: Thank  You for the code review I´ve removed the demo url default value, let me know if more changes are required :)

File: src/project-infos.spec.js [hunk 2]
```diff
@@ -42,6 +42,7 @@ describe('projectInfos', () => {
         author: 'Franck Abgrall',
         license: 'MIT',
         homepage: 'https://github.com/kefranabg/readme-md-generator',
+        demoUrl: 'https://github.com/kefranabg/readme-md-generator#-demo',
```
Comment (kefranabg) [inline src/project-infos.spec.js]: This line should be removed

File: src/project-infos.spec.js [hunk 3]
```diff
@@ -68,6 +69,7 @@ describe('projectInfos', () => {
         author: 'Franck Abgrall',
         repositoryUrl: 'https://github.com/kefranabg/readme-md-generator',
         homepage: 'https://github.com/kefranabg/readme-md-generator',
+        demoUrl: 'https://github.com/kefranabg/readme-md-generator#-demo',
```
Comment (kefranabg) [inline src/project-infos.spec.js]: This line should be removed

File: src/project-infos.js [hunk 4]
```diff
@@ -127,6 +127,7 @@ const getProjectInfos = async () => {
   const version = get(packageJson, 'version', undefined)
   const licenseName = get(packageJson, 'license', undefined)
   const homepage = get(packageJson, 'homepage', undefined)
+  const demoUrl = get(packageJson, 'demoUrl', undefined)
```
Comment (kefranabg) [inline src/project-infos.js]: To remove

File: src/project-infos.js [hunk 5]
```diff
@@ -152,6 +153,7 @@ const getProjectInfos = async () => {
     version,
     author,
     homepage,
+    demoUrl,
```
Comment (kefranabg) [inline src/project-infos.js]: To remove

File: src/readme.spec.js [hunk 6]
```diff
@@ -90,6 +90,7 @@ describe('readme', () => {
         'https://github.com/kefranabg/readme-md-generator#readme',
       projectHomepage:
         'https://github.com/kefranabg/readme-md-generator#readme',
+      projectDemoUrl: 'kefranabg',
```
Comment (kefranabg) [inline src/readme.spec.js]: You can use the url of this repos as an example

Comment (LucasProcopio) [reply src/readme.spec.js]: I have changed the `projectDemoUrl ` to this repos url

File: src/project-infos.js (hunk 5 above)
Comment (LucasProcopio) [reply src/project-infos.js]: Removed :)

File: src/project-infos.js (hunk 4 above)
Comment (LucasProcopio) [reply src/project-infos.js]: Removed :)

File: src/project-infos.spec.js (hunk 2 above)
Comment (LucasProcopio) [reply src/project-infos.spec.js]: Removed :)

File: src/project-infos.spec.js (hunk 3 above)
Comment (LucasProcopio) [reply src/project-infos.spec.js]: Removed :)

**Comment by kefranabg** [comment]
Thanks for your time!

@all-contributors please add @LucasProcopio for code

**Pull request merged**